src/database/queries/ApprovedItem.ts

```diff
@@ -32,12 +32,17 @@ export async function getApprovedItems(
   const baseArgs: prisma.Prisma.ApprovedItemFindManyArgs = {
     orderBy: { createdAt: 'desc' },
     where: constructWhereClauseFromFilters(filters),
+    include: {
+      authors: {
+        orderBy: [{ sortOrder: 'asc' }],
```

Comment:
Do we need to provide the `sortOrder` for **finding** and **deleting** items from the DB? The fact that we save them to the DB with the `asc` `sortOrder`, shouldn't the item be returned in the same sort order?

Unless it is a prisma thing where do find and delete, it needs to match the parameters of how it was saved?

Reply:
@Herraj, we definitely need to provide `sortOrder` when fetching a full `ApprovedItem` record. Even if records are written to a relational database sequentially, there is no guarantee they will be returned in that order unless you specify it explicitly. 

I am not sure about providing `sortOrder` when _deleting_ items. That shouldn't happen anywhere in the codebase.  The closest to this that I can think of is in the `deleteApprovedItem` mutation where we fetch the entire record, including the authors sorted correctly, _before_ we go ahead and hard delete everything - this is done to return the now deleted approved item as the mutation result for consistency with other mutations.